js/nested-for-loops.js

```diff
@@ -8,8 +8,17 @@ function nestedForLoops (depth, width = depth) {
   let result = "\n";
 
   // Write code here
-
+  for(let y=0; y<depth; y++){
+    
+    for(let x=0; x<width; x++){
+      let newStr = '{x:' + x + ', y:' + y + '}, ';
+      result+=newStr;
+    }
+    result = result.slice(0, -2);
```

Comment:
Although this perfectly removes the last `, ` from the result, it would be best to have some sort of comment explaining what's going on here above this line.